mta-sts.md

```diff
@@ -302,7 +302,7 @@ sender fetches a HTTPS resource (policy) from a host at `policy.mta-sts` in the
 Policy Domain. The policy is served from a "well known" URI:
 `https://policy.mta-sts.example.com/.well-known/mta-sts/current`. To consider 
```

Comment:
No, .well-known is a fixed string. See https://tools.ietf.org/html/rfc5785.

On Thu, Sep 8, 2016 at 8:41 PM Security Sauce <notifications@github.com>
wrote:

> In mta-sts.md
> <https://github.com/mrisher/smtp-sts/pull/95#discussion_r78065225>:
>
> > @@ -302,7 +302,7 @@ sender fetches a HTTPS resource (policy) from a host at `policy.mta-sts` in the
> >  Policy Domain. The policy is served from a "well known" URI:
> >  `https://policy.mta-sts.example.com/.well-known/mta-sts/current` <https://policy.mta-sts.example.com/.well-known/mta-sts/current>. To consider
>
> Is ".well.known" a keyword or a variable for something else?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mrisher/smtp-sts/pull/95/files/4d95a5bc2dc4403de0cb1559707d523ce3db60d4#r78065225>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1vixPL_VW66XJWrtpeGUakRrfk1MSJks5qoFb0gaJpZM4Jp_XD>
> .
>